src/NumericalAlgorithms/LinearOperators/ExponentialFilter.hpp

```diff
@@ -51,13 +51,15 @@ namespace Filters {
  * are desired for filtering different tensors, then multiple filters must be
  * inserted into the GlobalCache with different `FilterIndex` values. In
  * the input file these will be specified as `ExpFilterFILTER_INDEX`, e.g.
- * -  Filtering:
- * -    ExpFilter0:
- * -      Alpha: 12
- * -      HalfPower: 32
- * -    ExpFilter1:
- * -      Alpha: 36
- * -      HalfPower: 32
+ *```
```

Comment:
I would strongly prefer all examples to be snippets either from tests or input files

Reply:
Agreed. I've revised the unit test to test the parsing feature in question and the dox change is now a snippet.
Cheers!